static/app/constants/ios-device-list.tsx

```diff
@@ -4,6 +4,7 @@
 // and discard the rest of the JSON so we do not end up bloating bundle size.
 
 const iOSDeviceMapping: Record<string, string> = {
+  '': 'Siri Remote (2nd generation)',
```

Comment:
👀 is this part of some kind of build step? it also looks kinda wrong

Reply:
I keep letting it slip through because it keeps getting auto-generated 😠 I'm gonna ask FEDs what's going on